python-mini-projects/projects/Find_imdb_rating/find_IMDb_rating.py

```diff
@@ -0,0 +1,79 @@
+from bs4 import BeautifulSoup
+import requests
+import pandas as pd
+import os
+
+# Setting up session
+s = requests.session()  
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

**Problem**
This line of code might contain a resource leak. Resource leaks can cause your system to slow down or crash.

**Fix**
Consider closing the following resource: *s*. The resource is allocated by call *requests.session*. Execution paths that do not contain closure statements were detected. Close *s* in a try-finally block to prevent the resource leak.